src/test/eo/org/eolang/txt/text-tests.eo

```diff
@@ -445,3 +445,17 @@
         text " 漢Hello, dude  "
       0
     $.equal-to "漢"
+
+[] > temp-test
+  QQ.io.stdout > @
```

Comment:
@EugeneDar I think It will also be easier to understand the test works correctly if it would be `assert-that` here instead of `stdout`

Reply:
@levBagryansky it's garbage in the code, I forgot to delete it, I'll fix it now.